tests/test_dparray.py

```diff
@@ -32,3 +33,15 @@ def test_flatten(arr, arr_dtype):
     expected = numpy_array.flatten()
     result = dpnp_array.flatten()
     numpy.testing.assert_array_equal(expected, result)
+
+
+@pytest.mark.parametrize("shape",
+                         [(), 0, (0,), (2), (5, 2), (5, 0, 2), (5, 3, 2)],
+                         ids=['()', '0', '(0,)', '(2)', '(5, 2)', '(5, 0, 2)', '(5, 3, 2)'])
+@pytest.mark.parametrize("order",
+                         ["C", "F"],
+                         ids=['C', 'F'])
+def test_flags(shape, order):
+    expected = dpt.usm_ndarray(shape, order=order)
+    result = dpnp.ndarray(shape, order=order)
+    assert expected.flags == result.flags
```

Comment:
what if to expend the test (or add additional one), where to compare flags with numpy array? like:
```suggestion
    usm_array = dpt.usm_ndarray(shape, order=order)
    numpy_array = numpy.ndarray(shape, order=order)
    dpnp_array = dpnp.ndarray(shape, order=order)
    assert usm_array.flags == dpnp_array.flags
    assert numpy_array.flags.c_contiguous == dpnp_array.flags.c_contiguous
    assert numpy_array.flags.f_contiguous == dpnp_array.flags.f_contiguous
```